spec/spec.yaml

```
@@ -3936,6 +3952,180 @@ paths:
           schema:
             $ref: "./definitions.yaml#/definitions/V4GenericResponse"
 
+  /v5/clusters/{cluster_id}/labels/:
+    get:
+      operationId: getClusterLabels
+      tags:
+        - cluster labels
+      summary: Get cluster labels
+      description: |
+        Return labels attached to the supplied cluster.
+      parameters:
+        - $ref: "./parameters.yaml#/parameters/ClusterIdPathParameter"
+      responses:
+        "200":
+          description: Labels attached to this cluster
```

```suggestion
          description: Cluster labels
```

This shows up in a pretty small place in the docs ui, its describing the response.